server/perkeepd/ui/blob_item_audio.css

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Perkeep Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import (less) "prefix-free.css";
+
+
+.cam-blobitem-audio {
+	display: block;
+}
+
+.cam-blobitem-audio a {
+	text-decoration: none;
+}
+
+.cam-blobitem-audio .fa-volume-up {
+	color: #ccc;
+	display: block;
+	text-align: center;
+}
+
+.cam-blobitem-audio-play,
+.cam-blobitem-audio-pause {
+	color: rgba(125, 125, 125, 0.7);
+	position: absolute;
+	left: 50%;
+	top: 50%;
+	transform: translate(-50%, -50%);
```

Comment:
what does that do?

Reply:
`left: 50%` and `top: 50%` move the element's top left corner to the center `transform: translate(-50%, -50%)` moves the element 50% of it's width and height towards to the negative `x` and `y` direction.

In english, it centers the element using the element's center as an anchor

Reply:
ACK, thx.